### ISSUE TITLE
Fix missing code-owner alert for edx_arch_experiments.codejail_service.views

### ISSUE THREAD

**Issue by robrap** [edx/edx-arch-experiments#549]
This ticket is part inform, part implementation request.

I guess our team will be on-call for all changes in `edx_arch_experiments` at this time. I can't remember if we implemented a way to split ownership for something like this for enterprise, but it possibly isn't needed for this at this time anyway.

Since we are the default owners when there is no owner, fixing this won't do anything except quiet the alert that things there is a missing code-owner.

**Comment by robrap** [comment]
This may be a rare case. We'll just wait on alerting.
